src/pages/page-names.constants.ts

```diff
@@ -12,7 +12,7 @@ export const PASS_FINALISATION_PAGE = 'PassFinalisationPage';
 export const REKEY_SEARCH_PAGE = 'RekeySearchPage';
 export const TEST_REPORT_PAGE = 'TestReportPage';
 export const TEST_RESULTS_SEARCH_PAGE = 'TestResultsSearchPage';
-export const WAITING_ROOM_PAGE = 'WaitingRoomPage';
+export const WAITING_ROOM_PAGE = 'CatBWaitingRoomPage';
```

Comment:
This will need to be refactored. Open for suggestions on how to approach this

Reply:
I think if keep a standard convention for the format of the page name the Cat part is unnecessary... e.g if we said the format was <pagename><category> we could have WaitingRoomPageB, WaitingRoomPageBE, WaitingRoomC, with the category capitalised

Reply:
Maybe:

```
export const WAITING_ROOM_PAGE = {
  B: 'WaitingRoomPageB',
  BE: 'WaitingRoomPageBE',
  C: 'WaitingRoomPageC',
};
```
then use like:
```
this.navController.push(WAITING_ROOM_PAGE.B);
```

Reply:
Or perhaps:
```
export const CAT_B = {
  COMMUNICATION_PAGE: 'CommunicationPageB',
  WAITING_ROOM_PAGE: 'WaitingRoomPageB',
  WAITING_ROOM_TO_CAR_PAGE: 'WaitingRoomToCarPageB',
  };

export const CAT_BE = {
  COMMUNICATION_PAGE: 'CommunicationPageBE',
  WAITING_ROOM_PAGE: 'WaitingRoomPageBE',
  WAITING_ROOM_TO_CAR_PAGE: 'WaitingRoomToCarPageBE',
  };
```
then:
```
this.navController.push(CAT_B.WAITING_ROOM_PAGE);
```

Reply:
Thanks, will refactor this across the app for cat B in one go (PR after this one)

Reply:
> ```
> export const CAT_B = {
>   COMMUNICATION_PAGE: 'CommunicationPageB',
>   WAITING_ROOM_PAGE: 'WaitingRoomPageB',
>   WAITING_ROOM_TO_CAR_PAGE: 'WaitingRoomToCarPageB',
>   };
> 
> export const CAT_BE = {
>   COMMUNICATION_PAGE: 'CommunicationPageBE',
>   WAITING_ROOM_PAGE: 'WaitingRoomPageBE',
>   WAITING_ROOM_TO_CAR_PAGE: 'WaitingRoomToCarPageBE',
>   };
> ```

Just discussing with Szabi and I think I prefer this option.